sherpa/astro/ui/utils.py

```diff
@@ -6564,6 +6564,18 @@ def set_analysis(self, id, quantity=None, type='rate', factor=0):
         they are interpreted as the `id` and `quantity` parameters,
         respectively.
 
+        Sherpa does not match XSPEC when the `quantity` setting is set
+        to "channel", particularly when the data is grouped. Sherpa
+        retains the original CHANNEL values from the PHA dataset, so
+        calls to `ignore` and `notice` will use the same values even
+        if the grouping changes (although the results of these calls
+        may change depending on how the group edges match the selected
+        ranges), whereas XSPEC counts each group as a single channel.
+        This is also noticeable in calls to `plot_data`, `plot_model`,
+        and plot_fit`, where Sherpa will display each group with its
+        width in the number of PI or PHA channels, rather than having
+        a width of 1 as in XSPEC.
+
```

Comment:
I don't understand what a 'width of 1' means. 

We could just have one sentence at the end saying that Sherpa behaves differently than XSPEC and point to their documentation.

Reply:
As a user I might get confused because we haven't actually said what the difference is, and looking at the XSPEC documentation isn't going to help (in part because it doesn't really describe what it does, but in part because they are not going to have anything to say about what we do).

Reply:
Explaining the "width of 1" is hard to do (and at some level is the actual point of this paragraph). It's obviously a lot easier if we just don't add the text at all.

If you do care, you can see it above, in the XSPEC output, where each group has an effective width of 1 (as what XSPEC uses for channel numbers does not match the input channel grid), and is essentially the "group number", so each group has a width of 1 by default.For Sherpa we retain knowledge about the original channels (so the bins get placed correctly) an we divide by the number of channels in each group.